pkg/csaf/doc.go

```diff
@@ -0,0 +1,4 @@
+// Package csaf provides a library for Common Security Advisory Framework Version 2.0 (CSAF) documents.
```

Comment:
```suggestion
/*
Copyright 2023 The OpenVEX Authors
SPDX-License-Identifier: Apache-2.0
*/
// Package csaf provides a library for Common Security Advisory Framework Version 2.0 (CSAF) documents.
```

Reply:
I could be wrong, but we might also need an empty line between the boilerplate and the package doc comment, or else the boilerplate will get picked up in docs generation

Reply:
Verified we need to add a space after the copyright notice. I'll be sure to include a space.

Reply:
Fixed with https://github.com/openvex/go-vex/pull/7/commits/0ac3fdd5fd8a011049c64ea809b7e1a85b12de73